Sources/RunnerLib/Files Import/DangerFileGenerator.swift

```diff
@@ -15,7 +15,8 @@ public final class DangerFileGenerator {
             let url = importsRegex.replacementString(for: result, in: dangerContent, offset: 0, template: "$1")
 
             guard let fileContent = try? String(contentsOfFile: url),
-                let replacementRange = Range<String.Index>(result.range, in: dangerContent) else {
+                let replacementRange = Range<String.Index>(result.range, in: dangerContent)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/RunnerLib/Files Import/DangerFileGenerator.swift;
  Line: 11;
-->

- :warning: Line should be 120 characters or less: currently 140 characters (`line_length`)